torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -174,6 +174,53 @@ def affine_image_pil(
     return _FP.affine(img, matrix, interpolation=pil_modes_mapping[interpolation], fill=fill)
 
 
+def affine_bounding_box(
+    bounding_box: torch.Tensor,
+    format: features.BoundingBoxFormat,
+    image_size: Tuple[int, int],
+    angle: float,
+    translate: List[float],
+    scale: float,
+    shear: List[float],
+    center: Optional[List[float]] = None,
+) -> torch.Tensor:
+    original_shape = bounding_box.shape
+    bounding_box = convert_bounding_box_format(
+        bounding_box, old_format=format, new_format=features.BoundingBoxFormat.XYXY
+    ).view(-1, 4)
+
+    dtype = bounding_box.dtype if torch.is_floating_point(bounding_box) else torch.float32
+    device = bounding_box.device
+
+    if center is None:
+        height, width = image_size
+        center_f = [width * 0.5, height * 0.5]
+    else:
+        center_f = [float(c) for c in center]
+
+    translate_f = [float(t) for t in translate]
+    affine_matrix = torch.tensor(
+        _get_inverse_affine_matrix(center_f, angle, translate_f, scale, shear, inverted=False),
```

Comment:
Question: Can you talk more about why here is `inverted=False`?

Reply:
If we would like to transform points with an affine matrix we do something like `new_points = M * points` where `M` is affine (non-inverted) matrix. 
For images we needed inverse affine matrix as we compute pixels in reversed way:
```
for each out_x, out_y
    in_pts = InvM * out_pts
    output_image[out_pts] = interpolate(input_image, in_pts)
```
That's why now we need a method to get affine matrix and its inverse. We have a method to get inverse and now we need non-inverted one.